src/voting-metadata-api/.env.template

```diff
@@ -1,4 +1,4 @@
 NODE_ENV=development
 ALLOWED_ADDRESSES=any
-API_BINARY_CONTENT_TYPES=image/*
```

Comment:
Is this one still needed?

Reply:
Great question, couldn't find any resources on it and haven't run into it before

Reply:
I think it's this: https://github.com/dougmoscrop/serverless-http/blob/master/docs/ADVANCED.md#binary-mode

Admittedly, that means that this bit should be `BINARY_CONTENT_TYPES` - the `API_` is used in the cdk infrastructure